ruby/cross_service_examples/item-tracker/README.md

```diff
@@ -0,0 +1,28 @@
+# Item Tracker Cross-Sercice Example
+
+This example code comprises a "real-world" reference implementation of Amazon Relational Database Service (RDS) Aurora.
```

Comment:
You can just use Amazon Aurora on first use, or if you want to refer to RDS, the full name of RDS is Amazon Relational Database Service (Amazon RDS), but don't conflate the two.